stories/SetContactPropertyBlock.stories.ts

```diff
@@ -0,0 +1,40 @@
+import Vue from 'vue'
```

Comment:
Rather than editing builder.config.json, can we update the configuration explicitly here so we can test with example properties? I believe you should be able to update `stories/story-utils/FlowBuilderSidebarEditorContainer.vue` to accept builder and app config as props in the same way InteractionDesigner does.

Reply:
thanks for the proposal.  According the discussion Brett & I had, we'll try to not edit the `builder.config.json` as possible (unless it will improve devs' productivity), and will put any sample data in storybook side